### ISSUE TITLE
Remove entire desktop sharing

### ISSUE THREAD

**Issue by martinthomson** [w3c/mediacapture-screen-share#1]
Feedback from some folks at Mozilla is that sharing the entire desktop has some undesirable properties:
- It forces us to black out the browser to provide the protection that we need for CSRF
- It opens up the option for a click through dialog of the form "Do you want to share your desktop? y/n", which removes the implicit consent advantage of a forced selection.
- It provides challenges for feedback similar to those posed by a full-screen application 
- It isn't actually that desirable a feature

In short, we are unlikely to implement this particular feature in Firefox at this stage.

**Comment by martinthomson** [comment]
@keithgrif says:
Desktop share is quite commonly used and offers several advantages (and challenges). I dont understand the requirement to black out the browser and the CSRF protection concern, it would be great to understand that in more detail. We should also discuss the requirements and capabilities that constitute desktop share.

**Comment by martinthomson** [comment]
> I dont understand the requirement to black out the browser and the CSRF protection concern, it would be great to understand that in more detail.

No one does. That's the problem. Read https://tools.ietf.org/html/draft-ietf-rtcweb-security-07#section-4.1.1 for details. Unfortunately, you need a pretty strong grounding in the cross origin model to get this stuff.

**Comment by keithgrif** [comment]
I think we can close this one off after the last media capture WG meeting. The term "monitor" was adopted to describe a physical display surface and updated in the current revision.
